web/utilitis/saveData.js

```diff
@@ -0,0 +1,11 @@
+export const STORAGE_KEY = 'cvFormData';
+
+export const getFormData = () => {
+  return JSON.parse(localStorage.getItem(STORAGE_KEY)) || {};
+};
+
+export const saveFormData = (newData) => {
+  const existing = getFormData();
```

Comment:
hmmm i'd say that this is unnecessary as we should always be getting the form data before we save it. I would remove this and the updated below as this might be a place where tricky bugs might show up in the future